subprojects/core/src/integTest/groovy/org/gradle/api/tasks/TaskCommandLineConfigurationIntegrationSpec.groovy

```diff
@@ -225,7 +225,7 @@ class TaskCommandLineConfigurationIntegrationSpec extends AbstractIntegrationSpe
 
     def "single dash user error yields decent error message"() {
         when:
-        runAndFail 'help', '-task'
+        runAndFail 'help', '-tsk'
```

Comment:
:x: Why did this need to change?

Reply:
Because `-task` is equivalent to `-t -a -s -k` and `-a`(`--no-rebuild`) is exactly what we're deprecating. To avoid the deprecation warning, this is the simplest approach.

Reply:
OK, thanks.